os/net/routing/rpl-classic/rpl.h

```diff
@@ -262,6 +262,10 @@ struct rpl_instance {
   struct ctimer dao_lifetime_timer;
   struct ctimer unicast_dio_timer;
   rpl_parent_t *unicast_dio_target;
+  struct ctimer unicast_dao_timer;
+  rpl_parent_t *unicast_dao_target;
+  uip_ipaddr_t *unicast_dao_prefix;
+  uint8_t unicast_dao_lifetime;
```

Comment:
I'm guessing there will be padding between `unicast_dao_lifetime` and `dao_retransmit_timer`. Putting these fields after `dao_retransmit_timer` should avoid that.

Reply:
:+1: